docker/Dockerfile.ci

```diff
@@ -77,6 +85,22 @@ COPY environment.yml /root/code/garage/environment.yml
 RUN conda env create -f /root/code/garage/environment.yml && \
   rm -rf /opt/conda/pkgs/*
 
+# Roboschool in conda
+RUN git clone --depth 1 https://github.com/openai/roboschool.git && \
+  git clone --depth 1 https://github.com/olegklimov/bullet3 -b roboschool_self_collision && \
+  mkdir bullet3/build && \
+  cd bullet3/build && \
+  cmake -DBUILD_SHARED_LIBS=ON -DUSE_DOUBLE_PRECISION=1 \
+      -DCMAKE_INSTALL_PREFIX:PATH="../../roboschool/roboschool/cpp-household/bullet_local_install" \
+      -DBUILD_CPU_DEMOS=OFF -DBUILD_BULLET2_DEMOS=OFF -DBUILD_EXTRAS=OFF\
+      -DBUILD_UNIT_TESTS=OFF -DBUILD_CLSOCKET=OFF -DBUILD_ENET=OFF \
+      -DBUILD_OPENGL3_DEMOS=OFF .. && \
+  make -j"$(nproc)" && \
+  make install && \ 
+  cd ../..
+RUN ["/bin/bash", "-c", "source activate garage && pip install ./roboschool"]
```

Comment:
Is there a reason we can't make this all one layer (RUN command)? That's the convention in docker.